tests/test_verifylib.py

```diff
@@ -1023,7 +1032,17 @@ def test_verify_passing_double_signed_layout(self):
             [self.alice_path, self.bob_path]
         )
         in_toto_verify(layout, layout_key_dict)
-
+    
+    def test_verify_failing_layout_current_datetime(self):
+        """Test pass verification with current date expiry layout."""
+        with patch("""in_toto.verifylib.datetime""") as mock_date:
+            mock_date.datetime.now.return_value= datetime(2025, 1, 2, 3, 4, 6).replace(tzinfo=pytz.utc)
+            assert in_toto.verifylib.datetime.datetime.now() == datetime(2025, 1, 2, 3, 4, 6).replace(tzinfo=pytz.utc)
+            layout = Metablock.load(self.layout_single_signed_path)
+            layout_key_dict = import_publickeys_from_file([self.alice_path])
+            with self.assertRaises(LayoutExpiredError):
```

Comment:
Neat, I think we can also add another where the time is off by a second and the expiry check passes.